src/main/java/org/fundacion/jala/converter/models/ProjectSQL.java

```diff
@@ -13,14 +13,18 @@
 import javax.persistence.EntityManagerFactory;
 import javax.persistence.Persistence;
 
+/**
+ * This class is to query to database.
+ */
 public class ProjectSQL {
 
     private static EntityManagerFactory emf = Persistence.createEntityManagerFactory("jalaPersistence");
 
     /**
-     * This method is for to insert dates for database
-     * @param projectName name the archive
-     * @param pathProject path the archive
+     * This method is for to insert dates for database.
```

Comment:
The method should be described in third person. On the other hand, it should be data instead of date. Finally, "is for to" is not very clear, probably better to remove the for.

Reply:
ok